dev/documentation/en/datamodel-keyconcepts.md

```diff
@@ -0,0 +1,100 @@
+---
+book: dev-documentation
+version: 3.4
+title: Entities Key Concepts - Technical Documentation - OJS|OMP|OPS
+---
+
+# Entities: Key Concepts
+
+The overall database schema for the PKP data model consists of well over 100 tables.
+The present page does not introduce the entire data model.
+It merely characterizes a few particularly important entities
+and describes some other things that one might _expect_ to be entities
+but that happen to be represented in some other way.
+
+The page is organized as a loose list of sections in alphabetical order.
+
+When you look for the classes representing these entities,
+some of them will live in PKPlib, others in OJS (etc.),
+and many in both, with an inheritance relationship between them. 
+In the latter case, the superclass in PKPlib usually has a `PKP` name prefix.
+
+## Author
+
+Authors are associated with [Publications](#Publication).
+An Author need not be a [User](#User).
+Except for the email address, an Author's attributes such as
+names and affiliations can be localized.
+
+## Context
+
+Contexts appear _everywhere_ in PKP.
+In OJS, a Context represents a journal.
+Except for Plugins and Users, most things are per context, not shared between contexts.
+
+## Editor
+
+Editors as such are _not_ entities.
+They are Users that get assigned an Editor role in some scope such as
+an entire [Context](#Context) or a Section.
+
+## Publication
+
+Publications represent the various versions of a [Submission](#Submission)
+as they are created during the Submission's lifecycle, e.g. the
+version submitted for initial review, for second review, for publication,
+the version after copyediting, and the final published version.
+Publications would better be called SubmissionVersions (but are not).
```

Comment:
Although it makes sense at first, it's probably better not to think of publications as versions of submissions. Since we introduced versioning, there's a strong distinction between the object of publishing (eg - the publication, with title, metadata, contributors, etc) and the object the editorial workflow (eg - the submission, with assigned users, review assignments, submission files, etc).